Merge `db-tools:gdprify` and `db-tools:anonymize` for a more GDPR-friendly workflow (#75)
makinacorpus/DbToolsBundle: docs/content/anonymization/gdprify.md

# A GDPR-friendly workflow

The *DbToolsBundle* provides a command to run a *GDPR-friendly* workflow.

> The idea behind this command was to provide a way to import production data to another environment

Author: ```suggestion
The idea behind this command is to provide a way to import production data to another environment
```

while ensuring not keeping sensitive data.

After you got back a backup file from your production environment, run:

```sh
console db-tools:gdprify path/to/your/production/backup
```

This command successively:

1. **import** the given backup file,
2. **anonymize** the database,
3. **backup** the newly anonymized database.

::: warning
The last step of this workflow will **overwrite** the given backup file:
this way, no sensitive data remain on your disk.
:::

:::danger
It has to be noted that this workflow could be considered as problematic.

To stricly follow GDPR recommendations, **you should not get back a production backup

> file on you own computer**. Event if this backup file will quickly disappear from

Reviewer: ```suggestion
file on your own computer**. Even if this backup file will quickly disappear from
```

your disk. Sensitive data should not transit on your machine.

> So the next question is: why do we provide such a worflow ?

Reviewer: ```suggestion
So the next question is: why do we provide such a worflow?
```
